### DISCUSSION TITLE
Ajout d'un séparateur de milliers aux valeurs

### DISCUSSION THREAD

**Discussion by Syandre** [Crownicles/Crownicles#1791] [Idées acceptées / déjà implémentées]
Simple proposition d'intégrer le rajout du point, d'un espace ou de la virgule en tant que séparateur de milliers aux valeurs, afin d'en améliorer la lecture.

Par exemple, notamment lorsque un certain orateur annonce que "la banque du pays compte 106673687💰 en circulation [...]" .

Un affichage de 106.673.687 ; 106,673,687 ou 106 673 687 en simplifierait la lecture.

**Comment by BastLast** [comment]
Pas bête ça, je sais pas si ya un moyen izi de le faire auto sur les nombres ? @DraftBot-A-Discord-Adventure/contributors

**Comment by Syandre** [comment]
Après renseignements, une note supplémentaire pour question de propreté (et éviter de s'arracher les cheveux entre les normes des différents pays 😬) :

- En français on utilise l'espace ou le point comme séparateur de milliers. 
- En anglais, on utilise l'espace ou la virgule comme séparateur de milliers.

Pour reprendre les 106673687💰 donnés en exemple, ça donne :
FR: 106.673.687💰 ou 106 673 687💰. 
EN: 106,673,687💰 ou 106 673 687💰. 

Au choix entre utiliser l'espace pour tout le monde (du moins tant que le bot est FR/EN), ou d'intégrer l'un et l'autre format avec la traduction si c'est réalisable.

**Reply by BastLast** [reply]
après avoir fouillé un poil plus c'est pas super simple techniquement à faire et vu l'importance du truc, ya des chances qu'on fasse pas ça tout de suite. Mais c'est définitivement dans la todoliste à long terme

**Reply by BastLast** [reply]
je le passe en idée acceptée et je convertie cela en issue :)
